ui/app/components/tx-view.js

```diff
@@ -79,18 +97,25 @@ TxView.prototype.render = function () {
           style: {
             borderRadius: '25px',
             border: '1px solid',
-          }
+          },
         }),
 
         // laptop: 5vw?
         // phone: 50vw?
-        h('div.flex-column.flex-center', {
-          style: {}
-        }, [
-          h('div', {}, '1001.124 ETH'),
-
-          h('div', {}, '$300,000 USD'),
-        ]),
+        // h('div.flex-column.flex-center', {
```

Comment:
We can just delete these if we have the new data. 
See part 7 - we've moved to scss - so the text is being styled differently.

Takeaways:
- I suggest not leaving commented code when you're PRing feature work that replaces [presentational components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0).
- This area is now being styled differently, since Part 7. I suggest checking it out and making sure rendering with EthBalance looks right. If not, we'll need to refactor `EthBalance` (this would be a good time to add a few unit tests as well).